src/components/CertificateVerifyBlock/DetailedCertificateVerifyBlock.test.js

```diff
@@ -1,93 +1,102 @@
 import React from "react";
```

Comment:
it's a bit complicated to review this because of snapshot testing. For instance:

![image](https://user-images.githubusercontent.com/503185/66178516-3c253680-e698-11e9-8686-379b41e00dfa.png)

According to the test title it should render correctly but the content says the certificate has been tampered with. I would suggest we dont use snapshot testing and test the outputted text from the component

Reply:
remove snapshot tests.